.github/workflows/main.yml

```diff
@@ -45,3 +44,4 @@ jobs:
           rustup toolchain install 1.34.0
           rustup default 1.34.0
       - run: cargo build --all-targets --all
+
```

Comment:
stray line?

Reply:
Hmm, to be honest in my local copy there is no empty line at the end. Don't know why it shows up here.

Reply:
Fixed it.